index.js

```diff
@@ -38,10 +39,13 @@ Notifier.prototype.handler = function( request, response ) {
 			}
 			data = JSON.parse( data );
 		} catch( error ) {
+
 			// Invalid data, stop processing
+			var newError = new Error( "Invalid data parsing payload. " + error.message );
```

Comment:
Let's just reuse the `error` variable and reduce the wording:

`error = new Error( "Invalid data: " + error.message );`